WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -20,42 +21,77 @@ final public class PushNotificationsManager : NSObject
     
     /// Returns the shared PushNotificationsManager instance.
     ///
-    static let sharedInstance = PushNotificationsManager()
+    public static let sharedInstance = PushNotificationsManager()
```

Comment:
This class, and thus the methods, shouldn't need to be public.

Feel free to skip in this comment though, as the visibility of this class doesn't fall inside the scope of this PR.